src/interfaces/DestinationService.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  DeliveryResponse,
+  MetaTransferObject,
+  ProcessorTransformationRequest,
+  ProcessorTransformationResponse,
+  RouterTransformationRequestData,
+  RouterTransformationResponse,
+  ProcessorTransformationOutput,
+  UserDeletionRequest,
+  UserDeletionResponse,
+} from '../types/index';
+
+export default interface DestinationService {
+  init(): void;
+
+  getTags(
+    destType: string,
+    destinationId: string,
+    workspaceId: string,
+    feature: string,
+  ): MetaTransferObject;
+
+  processorRoutine(
```

Comment:
Did we discuss naming conventions for these?
routerRoutine -> doRouterTransform?
processorRoutine -> doProcessorTransform?

Reply:
Yes updated